### DISCUSSION TITLE
White space in control.

### DISCUSSION THREAD

**Discussion by timvz** [abbaye/WPFHexEditorControl#115] [Q&A] [answered]
I see a LOT of whitespace between the hex bytes and the vertical scrollbar.  Is there a way to make it so the control width auto-sizes to a width needed to display just the hex and the string data panel?

**Comment by abbaye** [comment, marked as answer]
Hi, 

you can adjust the size of the control 😉 

You can also use the build-in visibility property to adapt the control as you want to show.

- HexDataVisibility
- HeaderVisibility
- StringDataVisibility
- BarChartPanelVisibility
- StatusBarVisibility
- LineInfoVisibility
